### PR TITLE
Fix bugs found during Leipzig meetup

### PR DESCRIPTION
**Pull request by mgax**

### CODE REVIEW

File: ansible/roles/network/templates/network-manager-dnsmasq.conf [hunk 1]
```diff
@@ -0,0 +1 @@
+address=/{{ liquid_domain }}/10.42.0.1
```
Comment (gabriel-v) [inline ansible/roles/network/templates/network-manager-dnsmasq.conf]: Let's make a backlog issue for actually using the LAN ip config.

File: ansible/roles/network/tasks/main.yml [hunk 2]
```diff
@@ -12,3 +12,11 @@
   copy:
     src: network-interfaces/eth0
     dest: /etc/network/interfaces.d/eth0
+
+- name: dnsmasq configuration for network-manager
+  template:
+    src: network-manager-dnsmasq.conf
+    dest: /etc/NetworkManager/dnsmasq-shared.d/liquid
```
Comment (gabriel-v) [inline ansible/roles/network/tasks/main.yml]: Give this config file an extension, if it makes sense

File: ansible/roles/hypothesis/tasks/main.yml [hunk 3]
```diff
@@ -139,5 +139,3 @@
     - user-created
     - user-passwd
     - user-deleted
-  tags:
```
Comment (gabriel-v) [inline ansible/roles/hypothesis/tasks/main.yml]: So this one is actually fun. That script under https://github.com/liquidinvestigations/setup/blob/fix/ansible/roles/hypothesis/files/hooks/user#L7 still has some jinja template in it. It's only used for the email generation, so all the hypothesis users we generate are `username@{{ liquid_domain }}`.

Either move them back to templates and keep this tag, or go to that file and make all emails end in `@liquid.investigations.bogus.email.org` or something

File: ansible/roles/network/templates/network-manager-dnsmasq.conf (hunk 1 above)
Comment (mgax) [reply ansible/roles/network/templates/network-manager-dnsmasq.conf]: Here you go. https://github.com/liquidinvestigations/setup/issues/75

File: ansible/roles/hypothesis/tasks/main.yml (hunk 3 above)
Comment (mgax) [reply ansible/roles/hypothesis/tasks/main.yml]: Good catch, thanks.

**Pull request merged**